bin/docker-entrypoint.sh

```diff
@@ -15,6 +15,17 @@ export PGUSER=${DB_USER}
 export PGPASSWORD=${DB_PASSWORD}
 export PGDATABASE=${DB_NAME}
 
+# As docker-compose exec do not launch the entrypoint
```

Comment:
s/do not/does not/
May be you could also add the comment:
```The best would be to use directly the PG vars into the odoo template, but would be a breaking change```\
so eventually this is refactored better in the future.